cumulusci/tasks/preflight/licenses.py

```diff
@@ -23,3 +23,15 @@ def _run_task(self):
         ]
         licenses = "\n".join(self.return_values)
         self.logger.info(f"Found permission set licenses:\n{licenses}")
+
+
+class GetAvailablePermissionSets(BaseSalesforceApiTask):
+    def _run_task(self):
+        self.return_values = [
+            result["Name"]
+            for result in self.sf.query("SELECT Name FROM PermissionSet")[
```

Comment:
```suggestion
            for result in self.sf.query_all("SELECT Name FROM PermissionSet")[
```

query only returns the first page of results